Exercism/DifferenceOfSquares/DifferenceOfSquares.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public static class Squares
+{
+    public static double SquareOfSums(int max) => Math.Pow(Enumerable.Range(1, max).Sum(), 2);
```

Comment:
True, I just left it that way since that's how it came. I really need to start thinking of these as my own and only leaving their test class mostly untouched.